systems/robotInterfaces/@Biped/getLinkConstraints.m

```diff
@@ -64,7 +64,7 @@
 
 T = obj.getFrame(obj.foot_frame_id.right).T;
 % Torig * T = Tsole
-rfoot_sole_des = poseRPY2tform(rfoot_des) * T;
+rfoot_sole_des = tform2poseRPY(poseRPY2tform(rfoot_des) * T);
```

Comment:
yeah...

@gizatt and I had caught that on a branch he's been working on, but I'd failed to bring it into master until now